### DISCUSSION TITLE
Pre-configured NGINX reverse proxy with SSL + Let's Encrypt as a docker

### DISCUSSION THREAD

**Discussion by tomzorz** [Hubro/clipface#42] [Ideas]
Title basically says it all: the provided docker container is super nice to have, but could we get a second one that's pre-configured with the necessary bits for easily enabling SSL via Let's Encrypt?

**Comment by Hubro** [comment]
That sounds like it would be nice to have.

Do you know of any other Docker images that do this already? It would be nice to have a reference to start from.

**Comment by tomzorz** [comment]
Not really, although I can look around :)
